src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -92,9 +107,8 @@ class DependencyInfo extends React.Component<Props> {
       : getSvg.getWarningSvg();
 
     return (
-      <a
-        target={"_blank"}
-        href={downloadLink}
+      <div
+        onClick={() => openPrivacyModal(dependency)}
```

Comment:
Please, use role="button" or use button instead of div element
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role